Geometry/HGCalCommonData/src/HGCalProperty.cc

```diff
@@ -0,0 +1,29 @@
+#include "Geometry/HGCalCommonData/interface/HGCalProperty.h"
+
+const int32_t kHGCalFactor = 10;
```

Comment:
having these defined as globals in the global namespace is not preferred
better to put them inside the class/namespace

Reply:
Using namespace